charts/astronomer/values.yaml

```diff
@@ -227,7 +227,7 @@ houston:
     #   - 2.0.0-alpine3.10-onbuild
     #   channel: stable
 prisma:
-  podLabels: {}
+  podLabels: {} # This almost certainly does nothing. The right value is astronomer.podLabels
```

Comment:
@andriisoldatenko do you know anything about this?

Reply:
oh interesting your know what we are not using prisma pod anymore :) I assume we can get rid of it.

Reply:
Oh, so we can get rid of this whole prisma section then! Even better!